system/linux.nix

```diff
@@ -0,0 +1,139 @@
+{ lib, pkgs, ... }:
+
+let
+  keepmenu = pkgs.callPackage ../pkgs/keepmenu { };
+  rofimoji = pkgs.callPackage ../pkgs/rofimoji { };
+
+  wallpaper = lib.readFile (pkgs.callPackage ../scripts/wallpaper.nix { });
+
+  i3blocksConf = pkgs.callPackage ../config/i3blocks.nix { };
+  packages = [
```

Comment:
`packages = with pkgs; [` then you can remove `pkgs.` from the front of everything within the array

Reply:
thanks, makes sense.